server/@types/curiousApi/index.d.ts

```diff
@@ -265,7 +265,7 @@ export interface components {
   schemas: {
     /** AllAssessmentDTO */
     AllAssessmentDTO: {
-      v1?: components['schemas']['LearnerLatestAssessmentDTO'][]
+      v1?: components['schemas']['LearnerLatestAssessmentV1DTO'][]
```

Comment:
The changes in this file have all come from re-importing the Curious swagger spec
(it was a while ago that we first imported it, and their types for Assessments (and other things) have moved on quite considerably)